README.md

```diff
@@ -77,7 +77,8 @@ Open source projects that use `@faker-js/faker` as a dependency. You can go to t
 
 ## Languages
 
-- [C++ - faker-cxx](https://github.com/cieslarmichal/faker-cxx) - C++ Faker library for generating fake (but realistic) data. 
+- [C++ - faker-cxx](https://github.com/cieslarmichal/faker-cxx) - C++ Faker library for generating fake (but realistic) data.
+- [C#/F#/VB.NET - Bogus](https://github.com/bchavez/Bogus) - A port of Faker for .NET
```

Comment:
@ST-DDT I'm not sure about the sorting here. `+` would be equal char code 43 and `#` to 35. So techincally the entry `C#/F#/VB.NET` would come before `C++`? Like I said, I'm not sure.

Reply:
That is true.

`['C++','C#'].sort() -> [ "C#", "C++" ]`

I'm fine with both orders.

Reply:
```suggestion
- [C#/F#/VB.NET - Bogus](https://github.com/bchavez/Bogus) - A port of Faker for .NET
- [C++ - faker-cxx](https://github.com/cieslarmichal/faker-cxx) - C++ Faker library for generating fake (but realistic) data.
```